README.md

```diff
@@ -198,81 +197,72 @@ Example:
+The `liberty-plugin-archetype` is used to generate a basic multi-module project that builds a simple web application then deploys and tests it on a Liberty server. It also creates a Liberty server package that includes the application.
 
 #### Usage
 
     mvn archetype:generate \
         -DarchetypeGroupId=net.wasdev.wlp.maven \
         -DarchetypeArtifactId=liberty-plugin-archetype \
-        -DarchetypeVersion=2.1.2 \
-        -DwlpPluginVersion=2.1.2 \
+        -DarchetypeVersion=2.2 \
+        -DlibertyPluginVersion=2.2 \
         -DgroupId=test \
         -DartifactId=test \
         -Dversion=1.0-SNAPSHOT
 
 ### liberty-archetype-mp
 
-`liberty-archetype-mp` is used to generate a basic single-module project that builds a simple MicroProfile application then deploys and tests on a Liberty server. It also creates a minified, runnable Liberty server package that includes the application. The generated project includes [`liberty-maven-app-parent`](docs/parent-pom.md) parent pom that binds `liberty-maven-plugin` goals to the Maven default build lifecycle.
+The `liberty-archetype-mp` is used to generate a basic single-module project that builds a simple MicroProfile application then deploys and tests on a Liberty server. It also creates a minified, runnable Liberty server package that includes the application. The generated project includes [`liberty-maven-app-parent`](docs/parent-pom.md) parent pom that binds `liberty-maven-plugin` goals to the Maven default build lifecycle.
 
 #### Usage
 
     mvn archetype:generate \
         -DarchetypeGroupId=net.wasdev.wlp.maven \
         -DarchetypeArtifactId=liberty-archetype-mp \
-        -DarchetypeVersion=2.1.3 \
+        -DarchetypeVersion=2.3-SNAPSHOT \
         -DgroupId=test \
         -DartifactId=test \
         -Dversion=1.0-SNAPSHOT
         
-By default, the runtime used is the WebSphere Liberty MicroProfile package.  To change to use Open Liberty, you can set `-DlibertyGroupId=io.openliberty` and `-DlibertyArtifactId=openliberty-runtime`.  For example:
-
-    mvn archetype:generate \
-        -DarchetypeGroupId=net.wasdev.wlp.maven \
-        -DarchetypeArtifactId=liberty-archetype-mp \
-        -DarchetypeVersion=2.1.3 \
-        -DgroupId=test \
-        -DartifactId=test \
-        -DlibertyGroupId=io.openliberty \
-        -DlibertyArtifactId=openliberty-runtime \
-        -Dversion=1.0-SNAPSHOT
-
-By default, the `liberty-maven-plugin` version is set the same as the `liberty-archetype-mp` archetype version used. To specify a different version of the plugin, use the `wlpPluginVersion` parameter. For example, you could set `-DwlpPluginVersion=2.1`.
+For this archetype, you might want to use the `wlp-microprofile1` runtime. You can specify this by setting `-DruntimeGroupId=com.ibm.websphere.appserver.runtime` and `-DruntimeArtifactId=wlp-microprofile1`. 
```

Comment:
Is it defaulting to Open Liberty now? If so, should make that clear.

Reply:
That's at the top of the `Archetypes` section: 

```
By default, all archetypes that specify a Liberty runtime use the latest version of the Open Liberty runtime. 
```